include/rtdebug.h

```diff
@@ -87,7 +88,14 @@ while (0)
 #define RT_ASSERT(EX)                                                         \
 if (!(EX))                                                                    \
 {                                                                             \
-    rt_assert_handler(#EX, __FUNCTION__, __LINE__);                           \
+    rt_assert_handler(#EX, __FUNCTION__, __LINE__, RT_NULL);                  \
+}
+
+#define RT_ASSERT_MSG(EX, user_data, ...)                                     \
```

Comment:
```suggestion
#define RT_ASSERT_MSG(EX, fmt, ...)                                           \
```
建议改为 fmt 配合 args 的模式，并将其传入 rt_assert_handler